src/snowflake/snowpark/mock/plan.py

```diff
@@ -676,9 +685,10 @@ def describe(plan: MockExecutionPlan) -> List[Attribute]:
                 data_type = LongType()
             elif isinstance(data_type, FloatType):
                 data_type = DoubleType()
+
             ret.append(
                 Attribute(
-                    result[c].name,
+                    uppercase_and_enquote_if_not_quoted(result[c].name.strip()),
```

Comment:
Looks like `uppercase_and_enquote_if_not_quoted` and `strip` is always used together. Maybe we should put `strip` in `uppercase_and_enquote_if_not_quoted`?

Reply:
I changed to use `quote_name`, so I will keep the `strip()` here